### PR TITLE
GameINI: Add INIs for Brawl VC games

### PR DESCRIPTION
**Pull request by leoetlino**
Brawl ships with VC games that have their own ID and are different from the VC releases.

### CODE REVIEW

**Comment by leoetlino** [comment]
JP/PAL might use different IDs. In addition to that, JP has two extra VC games that aren't handled by this PR yet.

**Comment by JosJuice** [comment]
Should we really disable widescreen? Dolphin already shows these games using the correct aspect ratio for me even though my SYSCONF is set to 16:9.

**Comment by leoetlino** [comment]
I'm not sure, tbh (as I couldn't test). I just copied the values from the NES, SNES and N64 GameINIs. If it turns out to be unnecessary for all Brawl VC games, I can just remove that.

**Comment by gamemasterplc** [comment]
all you need to do is enable the wii's widescreen setting

**Comment by JosJuice** [comment]
Why was this closed? Getting the right settings for things like EFB copies is useful.

**Comment by leoetlino** [comment]
I deleted the wrong branch by mistake, which auto closed the PR. I didn't reopen it because I still needed to make some changes to it (the Widescreen stuff should be removed) and the PR is still missing the game IDs for the two extra games on the Japanese Brawl version.

**Comment by leoetlino** [comment]
Removed unneeded settings. I think we can merge this even if it lacks the Japanese game IDs -- we can always add them later.

**Comment by sepalani** [comment]
@leoetlino 
Does the Korean version of the game also have its own set of IDs? If so, does this PR lacks these?

**Comment by leoetlino** [comment]
I'm only aware of the Japanese version having two extra games.

If the Korean version also has extra games, then yes, this PR probably lacks information about them.

File: Data/Sys/GameSettings/HA9.ini [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# HA8xxx - Super Mario Bros. (Brawl VC)
```
Comment (xerxespersrex) [inline Data/Sys/GameSettings/HA9.ini]: Supposed to be HA9xxx to match the file name?

**Pull request merged**